python_modules/dagster/dagster/_core/remote_representation/external.py

```diff
@@ -87,6 +89,8 @@
     from dagster._core.scheduler.instigation import InstigatorState
     from dagster._core.snap.execution_plan_snapshot import ExecutionStepSnap
 
+_empty_set = frozenset()
```

Comment:
curious if this actually moves the needle to have a single one here, certainly can't hurt